src/resmom/linux/mom_mach.c

```diff
@@ -222,20 +225,21 @@ pbs_plinks	*Proc_lnks = NULL;	/* process links table head */
 static time_t	sampletime_ceil;
 static time_t	sampletime_floor;
 
+#if	MOM_CPUSET
 typedef int		pidcachetype_t;		/* type of allocatable unit */
 static pidcachetype_t	*pidcache_arena;	/* cache storage area */
 static unsigned int 	pidcache_bitsper = sizeof(pidcachetype_t) * NBBY;
 static int		pidcache_check(pid_t, pidcachetype_t *);
 static pidcachetype_t	*pidcache_create(void);
+static int		pidcache_reset(pidcachetype_t *);
 static void		pidcache_destroy(void);
-static int		pidcache_eligible(proc_stat_t *, char *, char *, unsigned long long);
 static pidcachetype_t *	pidcache_getarena(void);
```

Comment:
![screenshot from 2017-02-14 15-36-57](https://cloud.githubusercontent.com/assets/17924657/22924658/faef82be-f2cb-11e6-8968-e09192ce3a94.png)
this is how I am seeing it..